### DISCUSSION TITLE
How to use hybrid style in LAMMPS

### DISCUSSION THREAD

**Discussion by yyyien** [deepmodeling/deepmd-kit#2467] [Q&A]
I have installed the deepmd-kit using conda, and successfully trained a model locally. However, when I enter the following code :
```lammps
pair_style hybrid deepmd  frozen_model.pb lj/cut 7.5
pair_coeff * * 
pair_coeff * * lj/cut 0.006734447 3.166
```  
it leads to the following error:
```
ERROR: Pair hybrid sub-style is not used (src/pair_hybrid.cpp:568)
```

**Comment by njzjz** [comment]
See LAMMPS forum: https://matsci.org/t/error-pair-hybrid-sub-style-is-not-used/20541/2

**Reply by yyyien** [reply]
OK,thank you very much.
